godtools/App/Features/Tools/SwiftUI/Favorites/FavoritesContentViewModel.swift

```diff
@@ -70,7 +70,13 @@ class FavoritesContentViewModel: NSObject, ObservableObject {
         self.disableOptInOnboardingBannerUseCase = disableOptInOnboardingBannerUseCase
         
         super.init()
-        
+                
+        disableOptInOnboardingBannerSubscription = getOptInOnboardingBannerEnabledUseCase.getBannerIsEnabled()
+            .receive(on: DispatchQueue.main)
```

Comment:
So saying `receive(on: .main)` means you don't have to put `self?.hideTutorialBanner = !isEnabled` in an async block on main?  That's nice!

Reply:
Yeah much cleaner!

Reply:
My hope is too if we have multiple network requests happening in a UseCase being able to use Combine to merge those results will be much cleaner.